MapboxCoreMaps.podspec

```diff
@@ -22,6 +22,6 @@ Pod::Spec.new do |m|
 
   m.vendored_frameworks = 'MapboxCoreMaps.xcframework'
 
-  m.dependency 'MapboxCommon', '~> 21.0'
+  m.dependency 'MapboxCommon', '~> 21.1.0-beta'
```

Comment:
@macdrevx I wonder if we generate dependency correctly 🤔. Would `m.dependency 'MapboxCommon', '~> 21.1.0-beta'` work for common `21.1.0-beta.1` and `21.1.0-beta.2` releases? Is there an online tool for checking semver with optimistic operator?

Reply:
Hmm, I don't know if that would work, and I don't know of any tool. Our best bet would be to do a test.